jobs/build/appregistry/Jenkinsfile

```diff
@@ -166,19 +260,13 @@ node {
                         def token = retrieveBotToken()
                         for (def i = 0; i < operatorData.size(); i++) {
                             def build = operatorData[i]
-                            def metadata_nvr = buildlib.doozer("--group openshift-${params.BUILD_VERSION} operator-metadata:latest-build ${build.name}", [capture: true])
+                            def metadata_nvr = doozer("operator-metadata:latest-build ${build.name}")
```

Comment:
You might want to reuse `getMetadataNVRs` here.

Reply:
oh yes, forgot that one ... fixed: https://github.com/openshift/aos-cd-jobs/pull/2001/commits/b5a85fcb8c62267f3ded0e28491512e3716f0ec6